.gen/pipeline/api/openapi.yaml

```diff
@@ -18840,6 +18840,33 @@ components:
       - maxSize
       - minSize
       type: object
+    AWSAuthConfigMap:
+      properties:
+        data:
+          $ref: '#/components/schemas/AWSAuthConfigMap_data'
+      type: object
+    AWSAuthConfigMapUsers:
```

Comment:
I think these `mapRoles` and `mapUsers` (and`mapAccounts`, which is missing actually) should be mere strings. They are defined as strings in the original YAML format (Yaml strings).

Reply:
While they are indeed defined as multi-line strings on [AWS side](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html), it is not easy to define these on the REST API interface in the JSON, because the new lines are needed to be escaped as `\n`, which makes it much harder to specify it properly - with correct YAML-like multi-line string content which I believe is somewhat used manually as well (besides being used programatically).

I leave the trade-off decision for you (all), I just wanted to make sure it is a conscious decision.

Reply:
FYI: this is the generated descriptor, not the original one.

Reply:
Ah okay, so we offer the API in a structured way, in that way it makes sense to leave it like that.

Then the only thing left, please add mapAccounts as well.

Reply:
thx guys, I added `mapAccounts` field to the structure